FF1Blazorizer/Pages/Randomize.cshtml

```diff
@@ -463,7 +463,8 @@
 							<div class="checkbox-cell"></div>
 							<CheckBox UpdateToolTip="@UpdateToolTipID" Id="DisableTentSaving" bind-Value="@Flags.DisableTentSaving">Disable Tent/Cabin/House Saving on Overworld</CheckBox>
 							<CheckBox UpdateToolTip="@UpdateToolTipID" Id="DisableInnSaving" bind-Value="@Flags.DisableInnSaving">Disable Inn Saving</CheckBox>
-							<CheckBox UpdateToolTip="@UpdateToolTipID" Id="RandomizeFormationEnemizer" bind-Value="@Flags.RandomizeFormationEnemizer">Randomizer Formations (Enemizer)</CheckBox>
+							<CheckBox UpdateToolTip="@UpdateToolTipID" Id="RandomizeEnemizer" bind-Value="@Flags.RandomizeEnemizer">Randomize Enemies (Enemizer)</CheckBox>

```

Comment:
You can add sections like ` IsEnabled="@Flags.RandomizeEnemizer"` to all the checkboxes that are incompatible with the Enemizer. In other places you check and disable them but you can do it in the UI too so users understand why things they've checked off (like formation shuffle) won't work.

Reply:
Okay, I got that done.  I've also made it so that randomizing enemies requires selecting Generate New Formations first.  I've also renamed the options "Generate New Formations" and "Generate New Enemies" so that it is clear that this is not a standard shuffle.

The fun% flags will not be turned on/off since players store that information between seed generation, and I don't want to mess with that.